includes/Modules/Ads.php

```diff
@@ -219,13 +266,18 @@ protected function setup_settings() {
 	 * A module being connected means that all steps required as part of its activation are completed.
 	 *
 	 * @since 1.122.0
+	 * @since n.e.x.t Add additional check to account for paxConversionID and extCustomerID as well when feature flag is enabled.
 	 *
 	 * @return bool True if module is connected, false otherwise.
 	 */
 	public function is_connected() {
 		$options = $this->get_settings()->get();
 
-		return parent::is_connected() && ( ! empty( $options['conversionID'] ) || ! empty( $options['paxConversionID'] ) || ! empty( $options['extCustomerID'] ) );
+		if ( Feature_Flags::enabled( 'adsPax' ) ) {
+			return parent::is_connected() && ( ! empty( $options['conversionID'] ) || ! empty( $options['paxConversionID'] ) || ! empty( $options['extCustomerID'] ) );
+		}
+
+		return parent::is_connected() && ! empty( $options['conversionID'] );
```

Comment:
This method is really to return `false` if the module isn't connected. Then it returns the value of the parent method otherwise. See the implementations in other modules for reference.